ios/FluentUI/Command Bar/CommandBarItem.swift

```diff
@@ -125,7 +125,9 @@ open class CommandBarItem: NSObject {
     }
 
     /// Called after a property is changed to trigger the update of a corresponding button
-    var propertyChangedUpdateBlock: ((CommandBarItem) -> Void)?
+    /// - Parameter CommandBarItem: Instance of `CommandBarItem` the closure is being invoked from
+    /// - Parameter Bool: Indicates if the item's group state should also be updated
+    var propertyChangedUpdateBlock: ((_: CommandBarItem, _: Bool) -> Void)?
```

Comment:
i think because it isn't really apparent what this bool is for. lets actually have a variable? rather than _ ?

Reply:
I've added parameter names, however, Swift's type system requires that the parameters of function types are unnamed so the `_` before each parameter name is left in place. [This document](https://github.com/apple/swift-evolution/blob/master/proposals/0111-remove-arg-label-type-significance.md) provides some detail on why it works this way.

Reply:
thanks for the doc reference. can we do old school and in the places you call the propertychangedupateblock we do /* shouldUpdateGroupState*/ next do the boolean? i really think it isn't super apparant for the readers.